metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Provides the mysql2_chef_gem resource'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '1.1.0'
+version '1.2.0'
```

Comment:
In general you shouldn't bump the version of a cookbook in a PR. That's usually done during release time